logoapp/sp/js/sp_line_data.js

```diff
@@ -546,6 +546,51 @@ LineAppFramework.prototype.onClickUploadButton = function() {
 		URL.revokeObjectURL(url);
 	};
 	*/
+	clearInterval(this.completeLoopInterval	);
+	var uploadData = new Array(8);
+	for(var i=0; i<uploadData.length; i++) {
+		uploadData[i] = '';
+		for(var j=0; j<this.arrLineFrame[i].arrLine.length; j++) {
+			if(j!=0) {

```

Comment:
__[jshint]__
*  Use '!==' to compare with '0'.


<!-- Comment by [SideCI](https://sideci.com) -->